web/src/components/ak-textarea-input.ts

```diff
@@ -0,0 +1,58 @@
+import { AKElement } from "@goauthentik/elements/Base";
+
+import { TemplateResult, html, nothing } from "lit";
+import { customElement, property } from "lit/decorators.js";
+
+@customElement("ak-textarea-input")
+export class AkTextareaInput extends AKElement {
```

Comment:
Do this enough and you start to see two abstraction patterns, one around the content of the Input and one around the Layout.  I'll figure out both eventually.